packages/chakra-components/src/client.tsx

```diff
@@ -31,6 +31,8 @@ export const useClientProvider = ({ env: e, client: c, signer: s }: ClientProvid
 
     const opts: ClientOptions = {
       env: env as EnvOptions,
+      // TODO: REMOVE THE FOLLOWING csp_url when https://github.com/vocdoni/vocdoni-sdk/issues/163 is ready
+      csp_url: 'http://localhost:5000/v1',
```

Comment:
Should we merge this PR before or after this has been removed?

Reply:
hmmm I think best would be adding a condition to set this option in the SDK client (or not), if given. 
"CSP processes" using this set URL would work fine then, BUT all will use the same CSP server

Reply:
Well I think as a feature does not make much sense considering the CSP information is stored in the election metadata.

I'd wait until that PR and remove that part. BTW it should have been camelCased.

Reply:
Ok. 
hmmmm camelCased? You mean "csp_url"? If so, a ticket on the vocdoni-sdk should be opened, isn't it?

Reply:
Damn you're right, I didn't realize is an SDK option

Reply:
Issue created BTW https://github.com/vocdoni/vocdoni-sdk/issues/173